Testscripts/Linux/TestRDMA_MultiVM.sh

```diff
@@ -169,6 +174,43 @@ function Run_IMB_MPI1() {
 	fi
 }
 
+function Run_OMB_P2P() {
+	total_attempts=$(seq 1 1 $omb_p2p_tests_iterations)
+	omb_p2p_tests_array=($omb_p2p_tests)
+	for attempt in $total_attempts; do
+		LogMsg "OMB P2P test iteration $attempt for $mpi_type- Running."
```

Comment:
This log should be placed in the line 187 block. If it runs with non-mvapich mode, the logs do not make sense.

Reply:
done in revision